charts/mta-v7.x/INSTALL.md

```diff
@@ -16,12 +16,25 @@ helm install mta orchestrator-workflows/mta-v7.x -n sonataflow-infra
 ```
 
 ## Post-installation
-### Edit the `mta-analysis-v7-props` ConfigMap:
+### Edit the `${WORKFLOW_NAME}-creds` Secret
+The token for sending notifications from the MTA-v7.x workflow to RHDH notifications service needs to be provided to the workflow.
+
+Edit the secret `${WORKFLOW_NAME}-creds` and set the value of `NOTIFICATIONS_BEARER_TOKEN`:
+```
+WORKFLOW_NAME=mta-analysis-v7
+oc -n sonataflow-infra patch secret "${WORKFLOW_NAME}-creds" --type merge -p '{"data": { "NOTIFICATIONS_BEARER_TOKEN": "'$(oc get secrets -n rhdh-operator backstage-backend-auth-secret -o go-template='{{ .data.BACKEND_SECRET  }}')'"}}'
+```
+
+This secret is used in the `sonataflow` CR to inject the token as an environment variable that will be used by the workflow.
+
+Once the secret is updated, to have it applied, the pod shall be restarted. Note that if you are following the next section, no need for manual restart as any change to the sonataflow CR will restart the pod.
```

Comment:
same.

Since this is repeating itself I think we should have a guide for setting the notification token and then reference it. It will be useful for any workflow that wants to use notification.

Reply:
IMO we should remove all doc from repo at this point and only have it on the website and simply put a link to the README

With that, we should use the remoteMD tag to have common sections in separate pages but included in the same page to avoid links hell